Modules/Core/Common/CMakeLists.txt

```diff
@@ -201,10 +201,6 @@ set(ITK_USE_TBB ${Module_ITKTBB})
 configure_file(src/itkConfigure.h.in itkConfigure.h)
 configure_file(src/itkConfigurePrivate.h.in itkConfigurePrivate.h)
 
-if(ITK_WRAP_PYTHON)
```

Comment:
Hopefully this does not reintroduce errors from #5075, and other similar errors which #5030 was meant to address. I see there is a later commit 4784176870ea54a0fc18862a2e14aa79a34b8693 which deals with this.